Libraries/LibJS/Runtime/StringPrototype.cpp

```diff
@@ -319,6 +321,7 @@ JS_DEFINE_NATIVE_FUNCTION(StringPrototype::substring)
     if (interpreter.argument_count() == 0)
         return js_string(interpreter, string);
 
+    // FIXME: index_start and index_end should index a UFF-16 codepoint view of the string.
```

Comment:
Typo, UFF-16

Reply:
It's German: https://en.wiktionary.org/wiki/uff ...and apparently Norwegian: https://en.wikipedia.org/wiki/Uff_da